<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameCamera</class>
 <widget class="QFrame" name="FrameCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="LB_DisplayMat">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="PB_Play">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/ic_play_arrow_white_2x.png</normaloff>:/icons/ic_play_arrow_white_2x.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PB_Stop">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/ic_stop_white_2x.png</normaloff>:/icons/ic_stop_white_2x.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="CB_FPS">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>10 fps</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20 fps</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>30 fps</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>40 fps</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>50 fps</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>62 fps</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="CB_MatSize">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>640x480</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1280x720</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="CB_MatType">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="maxVisibleItems">
          <number>2</number>
         </property>
         <property name="maxCount">
          <number>10</number>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
         </property>
         <item>
          <property name="text">
           <string>Gray image</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RGB image</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="LB_DisplayConfig">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
